util/kubectl_test.go

```diff
@@ -0,0 +1,10 @@
+package util
+
+import "testing"
+
+func TestCheckKubectl(t *testing.T) {
+	result := CheckKubectl()
```

Comment:
Ah yes I just noticed that you're using Travis not Circle! That will definitely work then as I've installed packages on Travis before :)